a.core_concepts.md

```diff
@@ -175,7 +175,7 @@ kubectl get po -A
 <p>
 
 ```bash
-kubectl run nginx --image=nginx --restart=Never --port=80
+kubectl run nginx --image=nginx --restart=Never --expose --port=80
```

Comment:
--expose will create a Service. That's not part of the question, maybe we need to change the question?

Reply:
maybe keep both commands so show different ways.

Reply:
yup, we can accept a PR with that